WireUI/Sources/WireIndividualToTeamMigrationUI/Resources/Localizable.xcstrings

```diff
@@ -222,7 +222,7 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Accept our [Terms & Conditions](https://wire.com/en/terms-of-use-business) and [Privacy Policy](https://wire.com/en/privacy-policy)."
+            "value" : "Accept our [Terms & Conditions](https:\/\/wire.com\/en\/terms-of-use-business) and [Privacy Policy](https:\/\/wire.com\/en\/privacy-policy)."
```

Comment:
@El-Fitz urls should not be part of localizable, so we should have something like: 
```suggestion
            "value" : "Accept our [Terms & Conditions](%@) and [Privacy Policy](%@)."
```
Check Bundle+WireURLs.swift:52 and url.json

Reply:
Ok. But no need to escape the / in there.